xml/html_title.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fingerprints matches="http.html_title" protocol="http" database_type="service" preference="0.90">
```

Comment:
My preference for `matches` would be that it either doesn't exist (and recog uses the file name) or that it matches the file name sans `.xml`.